### DISCUSSION TITLE
动态域名解析，IP变动后不能及时更新

### DISCUSSION THREAD

**Discussion by lutao114** [IrineSistiana/mosdns#551] [General]
感谢作者的付出

  你好，我用的家庭宽带，有动态公网IP，做了动态域名解析，在IP发生变化后，域名解析的IP已经更新了，但是mosdns的解析缓存不能及时更新，导致无法访问，请问是否能实现某个域名的解析结果不缓存，每次访问都重新请求，或者针对我的问题有没有好的解决办法？  
   万分感谢！

**Comment by z499738316** [comment]
我也有同样的我问题，又方法解决吗

**Reply by lutao114** [reply]
你用的什么路由器？ 路由器设置端口转发的时候，有的路由器自动配置回流，即使你再局域网用域名访问，也会直接跳转到内网IP，有的不带回流，就会这样。  你可以直接在电脑上设置DNS劫持，或者在路由器上设置劫持，就能保证在局域网访问这个域名的时候直接解析到内网主机上
